### ISSUE TITLE
Options orders with command 'o' 

### ISSUE THREAD

**Issue by klepsydra** [anilshanbhag/RobinhoodShell#12]
Am I missing something, or are Option Orders not yet displayed with the 'o' command? 
('l' shows both stocks and options positions, great!)

**Comment by anilshanbhag** [comment]
Listing open option orders is not supported currently.

**Comment by nkaliape** [comment]
Anil, is this limitation of Robinhood API?

**Comment by anilshanbhag** [comment]
No, its not. However, its hard to type in an exact option order - so I just do it on mobile/webapp.

**Comment by murdrae** [comment]
Will this be added in the future?

**Comment by Emnolope** [comment]
+1 on this, I also want a way of doing options.

**Comment by jmmadsen** [comment]
@anilshanbhag I am working on creating my own fork with options. Can you expand on why you chose to not implement option orders? Do you view it as riskier programmatically and more error prone?

**Comment by anilshanbhag** [comment]
You need a good UI to choose the right strike / strike date. Also easy to do mistakes - for example if instead of typing in 0.05 you type 5, you will likely lose quite some money as most options are high bid/ask spreads.
